app/views/provider_interface/application_choices/_application_choice_header.html.erb

```diff
@@ -0,0 +1,16 @@
+<% if @application_choice.status == 'awaiting_provider_decision' -%>
+  <div class="app-banner">
+    <div class="app-banner__message">
+      <h2 class="govuk-heading-m govuk-!-margin-bottom-0">
+        You have <%= days_to_respond_to(@application_choice) %> days to send a response. <%= govuk_link_to 'Respond to application', provider_interface_application_choice_respond_path(@application_choice) %>.
+      </h2>
+    </div>
+  </div>
+<% end -%>
+
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-1">
+  <%= @application_choice.application_form.full_name %>
+  <%= render(ProviderInterface::ApplicationStatusTagComponent.new(application_choice: @application_choice)) %>
+</h1>
+
+<%= render SubNavigationComponent.new(items: @sub_navigation_items) %>
```

Comment:
Should we render this if `@sub_navigation_items.count == 1`?

Reply:
I'll add a `>1` conditional.